CrazyCanvas/engine_config_sandbox.json

```diff
@@ -6,7 +6,7 @@
   "FixedTimestep": 60,
   "NetworkPort": 4444,
   "RayTracingEnabled": true,
-  "MeshShadersEnabled": false,
+  "MeshShadersEnabled": true,
```

Comment:
Is this intentional?

Reply:
Yes, since the CrazyCanvas project uses MeshShaders I think Sandbox should use it too. They will automatically be disabled anyway if they aren't supported.